seatunnel-connectors-v2/connector-cdc/connector-cdc-base/src/main/java/org/apache/seatunnel/connectors/cdc/debezium/row/SeaTunnelRowDebeziumDeserializeSchema.java

```diff
@@ -184,7 +184,7 @@ private void deserializeDataChangeRecord(SourceRecord record, Collector<SeaTunne
             delete.setRowKind(RowKind.DELETE);
             delete.setTableId(tableId);
             collector.collect(delete);
-        } else {
+        } else if (operation == Envelope.Operation.UPDATE) {
```

Comment:
In addition to update related statements, are there any other statements that can reach the else block?

Reply:
TRUNCATE or MESSAGE

Reply:
Yes, we should ignore it. cc @hailin0